src/Stratis.Bitcoin/P2P/Protocol/Message.cs

```diff
@@ -135,14 +142,9 @@ public void ReadWrite(BitcoinStream stream)
         // FIXME: protocolVersion is not used. Is this a defect?
         private byte[] GetPayloadBytes(ProtocolVersion protocolVersion, out int length)
         {
-            MemoryStream ms = this.buffer == null ? new MemoryStream() : new MemoryStream(this.buffer);
+            MemoryStream ms =  new MemoryStream();
```

Comment:
Should we not put this in a `using`? Or is the stream used elsewhere still?

Reply:
solid will change